unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore_ECS7.cs

```diff
@@ -1,11 +1,13 @@
+using System.Collections.Generic;
 using DCL.Controllers;
 using DCL.ECSRuntime;
 
 namespace DCL
 {
     public class DataStore_ECS7
     {
-        public BaseDictionary<IParcelScene, ECSComponentsManager> componentsManagers = new BaseDictionary<IParcelScene, ECSComponentsManager>();
+        public BaseDictionary<IParcelScene, IECSComponentsManager> componentsManagers = new BaseDictionary<IParcelScene, IECSComponentsManager>();
+        public readonly BaseRefCountedCollection<(string sceneId, object model)> pendingSceneResources = new BaseRefCountedCollection<(string sceneId, object model)>();
```

Comment:
This makes really hard to find all the blockers for a sceneId.

It would be better to use:
```csharp
public readonly Dictionary<string, BaseRefCountedCollection<object>> pendingSceneResources = new Dictionary<string, BaseRefCountedCollection<object>>();
```

Also I'm not a big fan of having an `object` there, maybe would it be better to have a componentId or something?

Reply:
We need the model here so we can know exactly which resource is pending (imagine having different images, we need to know which one is loaded or pending) . If we want to remove the object here, we need to implement something like `IResource ` and implement it to every resource that is in the project so we can track the model of the resource. I thought it was an overengineer for the purpose